data/seeds/003_children.js

```diff
@@ -0,0 +1,13 @@
+exports.seed = function (knex) {
+  // Deletes ALL existing entries
+  return knex('table_name')
+    .del()
+    .then(function () {
+      // Inserts seed entries
+      return knex('table_name').insert([
+        { parent_id: 2, user_id: 5, username: 'FidgetSpinnersRLyfe', age: 12 },
+        { parent_id: 5, user_id: 20, username: 'FortniteGamerBoy', age: 13 },
+        { parent_id: 3, user_id: 34, username: 'VapeGod360XXX', age: 11 },
```

Comment:
Not to be a buzz kill of anything but we sure we wanna add this to the main branch. Just thinking in terms of the stakeholder. If it's not going to be shown to the stakeholder and won't show up on the frontend then ignore this message :)

Reply:
Fair. We'll fix that. Better safe than sorry